crates/sui-core/src/authority.rs

```diff
@@ -2374,39 +2375,6 @@ impl AuthorityState {
         Some(ChainIdentifier::from(*checkpoint.digest()))
     }
 
-    pub fn get_move_object<T>(&self, object_id: &ObjectID) -> SuiResult<T>
```

Comment:
These functions are not used anywhere, so I deleted them. lmk if we should still keep these